itkwidgets/_type_aliases.py

```diff
@@ -7,10 +7,9 @@
 from .integrations.pytorch import HAVE_TORCH
 from .integrations.vtk import HAVE_VTK
 from .integrations.xarray import HAVE_XARRAY
-from typing import Dict, List, Literal, Union
+from typing import Dict, List, Literal, Union, Tuple
 
-Gaussian_Curve = Dict[str, float]
-Gaussians = Dict[str, List[Gaussian_Curve]]
+Points2d = List[Tuple[float, float]]
```

Comment:
This could be

```
Sequence[Sequence[float, float]]
```

? Please check if variants work by passing them to the `set_*` method. If they do not work, then it may be worthwhile type casting with `list()` or `tuple()` to provide a more convenient interface for the caller.

In the documentation example there is:

```
[[.2, .1], [.8, .9]]
```

which is a List[List[float, float]].

Reply:
Ah, yes.  Updated type to `Sequence[Sequence[float]]`.  Sequence can't do `Sequence[float, float]`.  Passing tuple points or list points works.  Passing 3D point bugs out TF editor =/